server/src/main/java/io/druid/client/cache/CacheMonitor.java

```diff
@@ -38,20 +39,30 @@ public CacheMonitor(
     this.cache = cache;
   }
```

Comment:
if they are binding via guice then this shouldn't break anything.
I would remove it.
It is up to you though, I'm fine either ways :)